### PR TITLE
Simplify execution header proof

### PR DESCRIPTION
**Pull request by claravanstaden**
For execution header imports, we used to send the whole beacon block body, so that we could verify that the block body hash tree root is what we expect it to be, so that we know the Execution Payload is valid. However, now that that FastSSZ Go lib can generate proofs, we only have to send the Execution Payload and a proof. Then we can leave out the Beacon Block from the beacon client entirely.

### CODE REVIEW

File: relayer/relays/beacon/header/syncer/api/api_response.go [hunk 1]
```diff
@@ -515,6 +556,302 @@ func (b BeaconBlockResponse) ToScale() (scale.BeaconBlock, error) {
...
+	}
+
+	n := new(big.Int)
+	n, ok := n.SetString(executionPayload.BaseFeePerGas, 10)
+	if !ok {
+		return nil, err
+	}
+
+	// FastSSZ expects a little endian byte array
+	baseFeePerGas := util.ChangeByteOrder(n.Bytes())
```
Comment (yrong) [inline relayer/relays/beacon/header/syncer/api/api_response.go:691]: Interesting, Is this  because `baseFeePerGas` is actually a `uint256` in spec?

**Review by yrong — APPROVED** [review]
Awesome! Then I'll continue #786 based on this PR.

File: relayer/relays/beacon/header/syncer/api/api_response.go (hunk 1 above)
Comment (claravanstaden) [reply relayer/relays/beacon/header/syncer/api/api_response.go:691]: Yes, exactly! 😄 And in the spec unsigned ints are encoded to a 32 little endian byte, and big Int seems to use big endian. A bit of a pain, but we can remove this when we can convert the Lodestar block ssz bytes directly to the FastSSZ structs.

**Comment by claravanstaden** [comment]
> Awesome! Then I'll continue #786 based on this PR.

Great! I'm going to merge it to unblock you, and then I'll open up a new PR if @vgeddes, @alistair-singh or @doubledup have comments.

**Pull request merged**